packages/appstore/src/components/modals/real-wallets-upgrade/components/modal-elements/mobile-real-wallets-upgrade/mobile-real-wallets-upgrade.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { MobileDialog, ProgressBarTracker, SwipeableWrapper } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { TRealWalletsUpgradeSteps } from 'Types';
+import WalletSteps from '../wallet_steps';
+import './mobile-real-wallets-upgrade.scss';
+
+const MobileRealWalletsUpgrade = observer(({ wallet_upgrade_steps }: TRealWalletsUpgradeSteps) => {
+    const { traders_hub: is_real_wallets_upgrade_on } = useStore();
+
+    const wallet_steps = WalletSteps(wallet_upgrade_steps);
+    const { current_step, handleBack, handleNext } = wallet_upgrade_steps;
+
+    return (
+        <MobileDialog
+            portal_element_id='modal_root'
```

Comment:
I don't think you have to define `portal_element_id` everytime you used the `<MobileDialog/>` can you check if it is required if no then just ignore this comment 😅